src/rabbit_mgmt_wm_static.erl

```diff
@@ -21,8 +21,7 @@
 
```

Comment:
I think this entire file is not necessary anymore. Cowboy 2.0's cowboy_static supports reading files from .ez archives thanks to @dumbbell's patch. See https://git.ninenines.eu/cowboy.git/commit/?id=2166733628bbab0eb77eeed58bdf204727c48ec6

Reply:
This module uses this feature. Most of the code handles routing logic for management plugin applications. Is there a better way to do that?

Reply:
Nevermind I was reading the code too fast.